### PR TITLE
Integrated Enphase Script

### PR DESCRIPTION
**Pull request by aivantg**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

Added enphase utility function to pull in the production data for a particular system and user and called it from the index.js file

Added variables to environment variable file. 

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc"

### Next steps

Use this file to compute other data

CC: @dfangshuo

### CODE REVIEW

**Comment by aivantg** [comment]
Committed API keys, but reopening after removing them 😅

**Comment by aivantg** [comment]
I committed the api keys....again...reopening...again

File: utils/enphase.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const fetch = require('node-fetch'); // Import fetch (for network requests)
+const moment = require('moment');
+
+// SEE ENPHASE API DOCS HERE https://developer.enphase.com/docs
+// Returns object with `production`, `meter_production`, `micro_production` keys with production data
+const getEnphaseData = async (userId, systemId, startDate, endDate) => {
+  // Build API Url
+  const baseUrl = 'https://api.enphaseenergy.com/api/v2';
+  const key = process.env.ENPHASE_KEY;
```
Comment (dfangshuo) [inline utils/enphase.js]: nit: should we move this to constants to the top of the file or to a new constants file? 

i personally think that moving it to a new constants file might be too heavy of an approach at this point, but maybe moving it to the top?

File: index.js [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-}
+const EnphaseUtils = require('./utils/enphase.js');
 
-// Add other functions below that perform the individual operations
-// Example: (Pull data from EnPhase API, Generate PDF given billing data, etc.)
+async function main() {
+  console.log('Testing Enphase Data');
+  const result = await EnphaseUtils.getEnphaseData(
+    process.env.SAMPLE_ENPHASE_USER_ID,
+    process.env.SAMPLE_ENPHASE_SYSTEM_ID,
```
Comment (dfangshuo) [inline index.js]: is there any (dummy) inputs for `SAMPLE_ENPHASE_USER_ID` and `SAMPLE_ENPHASE_SYSTEM_ID` i can call this function with just to verify functionality?

File: utils/enphase.js (hunk 1 above)
Comment (aivantg) [reply utils/enphase.js]: Sure sure

File: index.js (hunk 2 above)
Comment (aivantg) [reply index.js]: Posted a sample environment file in slack!

**Pull request merged**